README.md

```diff
@@ -76,27 +53,6 @@ node make status
 tail -f ./logs/server.log
```

Comment:
It's not, but that's actually a bug. When I setup the task in `make.js` I forgot to have it log locally. It currently logs to forevers default location which is accessible via `node ./node_modules/.bin/forever logs app.js`.